### DISCUSSION TITLE
altstore  can't install uyouplus v17.49.6-2.1

### DISCUSSION THREAD

**Discussion by chenchenfang** [qnblackcat/uYouPlus#821] [Q&A]
when i install it,progress bar can't finish.
and just ask me app extensions keep or remove,
iphone 13 pro ios16.1
![IMG_F52E9B971E90-1](https://user-images.githubusercontent.com/50065243/209938286-ce3b645f-49d5-4e76-867e-1dcc38512d36.jpeg)

**Comment by pha1n0q** [comment]
Be patient- from my experience it always takes a long time and could take up to a couple of minutes. Be sure to leave AltStore open.

**Reply by chenchenfang** [reply]
> Be patient- from my experience it always takes a long time and could take up to a couple of minutes. Be sure to leave AltStore open.

OK I'll try again

**Reply by chenchenfang** [reply]
thans,when i wait about 5 minutes,the installation is complete.
